portafly/src/i18n/locales/en/audience/account/account.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Accounts > Account overview
+
+page_title: Accounts
+page_title_description: Page title of the subsection for Accounts Overview
+
+# Top area
+
+body_title: "Account: {{organization_name}}"
+body_title_description: Body title of the page for Accounts Overview
+
+# Tabs
+
+overview_title: Overview
+overview_description: This is the title of the tab containing the organization or group name, the administrator name/email address, signup date, and state.
+applications_title: Applications
+applications_description: This is the title of the tab containing the list of applications on the account, their names, the associated product, its plan, creation date, and its current state.
+
```

Comment:
@porueesq @dfennessy what about adding aria labels for tabs too?

Reply:
@cntlsn @josemigallas Just to confirm, can't HTML reuse the strings for tabs?

Reply:
I don't follow, wdym HTML? We can reuse any strings, provided they have the same meaning (same value) and they'll have the same translation. I've always been more inclined to have unique strings for everything because you can't guarantee they're gonna have the same value in a different language. But that's just me being cautious.

Reply:
Then yes, we can reuse everything.